src/components/Book.js

```diff
@@ -1,29 +1,33 @@
 import React from 'react';
 import { useDispatch } from 'react-redux';
+import { removeBook } from './redux/books/booksReducer';
 
 const Book = (book) => {
   const dispatch = useDispatch();
   const removeBookHandler = (e) => {
     const keyValue = e.target.value;
-    dispatch({ type: 'REMOVE_BOOK', payload: { key: keyValue } });
+
+    dispatch(removeBook(keyValue), []);
   };
 
   const { props } = book;
+  /* eslint-disable camelcase */
   /* eslint-disable react/prop-types */
```

Comment:
- [ ] Please avoid disabling Linters. They are used to flag programming errors. Fix those issues.